### ISSUE TITLE
CoreBluetoothMock is not possible to use with CBPeripheralManager or CBPeripheralManagerDelegate

### ISSUE THREAD

**Issue by mbt-laurent-n** [NordicSemiconductor/IOS-CoreBluetooth-Mock#40]
Hi all,
I use an application which use `CBPeripheralManager` or `CBPeripheralManagerDelegate`. 
Unfortunately this great library doesn't seem to support them. 
How could I use it with the CoreBluetoothTypeAliases method?

**Comment by philips77** [comment]
That's true. The library doesn't support `CBPeripheralManager` yet. Adding this feature is in our pipeline, but I don't see it coming this year.

**Comment by mbt-laurent-n** [comment]
Ok @philips77  thanks for confirming this and giving me the information.
Looking forward to test this feature :)

**Comment by philips77** [comment]
I'll keep it open as a reminder of the feature request.
